lib/services/presets.go

```diff
@@ -73,6 +73,7 @@ func NewPresetEditorRole() types.Role {
 					types.NewRule(types.KindDatabaseCertificate, RW()),
 					types.NewRule(types.KindInstaller, RW()),
 					types.NewRule(types.KindDevice, append(RW(), types.VerbCreateEnrollToken, types.VerbEnroll)),
+					types.NewRule(types.KindInstance, RW()),
```

Comment:
What's the point of RW on `instance` for a user that's not actively trying to do crimes in the backend? We don't default to RW on nodes for the editor role, for example.